_maps/templates/pirate_ship.dmm

```diff
@@ -664,6 +664,7 @@
 	name = "Pirate Ship";
 	port_direction = 8;
 	preferred_direction = 1;
+	timid = 0;
```

Comment:
wait no, this is probably no bueno now that i remember.

https://github.com/tgstation/tgstation/blob/b6149d7e59c7b500789e0536e038ea11c18fe146/code/modules/shuttle/shuttle.dm#L271-L273

Reply:
That comment refers to *shuttle templates*, as in those which will be loaded by the shuttle manipulator. This is just a template which happens to contain a shuttle.